gothemis/message/message.go

```diff
@@ -78,10 +78,16 @@ const (
 
 // Errors returned by Secure Message.
 var (
+	ErrEncryptMsg        = errors.New("Failed to encrypt message")
+	ErrDecryptMsg        = errors.New("Failed to decrypt message")
+	ErrSignMsg           = errors.New("Failed to sign message")
+	ErrVerifyMsg         = errors.New("Failed to verify message")
+	ErrProcessMsg        = errors.New("Failed to process message")
```

Comment:
I guess we can drop `Msg` from here entirely. (Or spell it `Message`, but I guess that's as redundant as `Data` in Secure Cell's `ErrEncryptData`.)